### ISSUE TITLE
JBoss improvement

### ISSUE THREAD

**Issue by Hipapheralkus** [augustd/burp-suite-software-version-checks#50]
Hi,
I got 200:ok page with following source:

```
<html><head><title>JBoss Web/7.4.10.Final-redhat-1 - JBWEB000064: Error report</title><style>
...
<h3>JBoss Web/7.4.10.Final-redhat-1</h3>

```
It is a bit different than the regexp in rules, and was not detected.
Thanks:)

**Comment by augustd** [comment]
Thanks! Make sure you reload the match rules in the extension UI. 

Keep 'em coming
